AsstIntegration

```diff
@@ -153,21 +168,30 @@ start_Plant() {
 # Startup for Google Assistant Env + Velostat Calendar
 start_Calendar() {
   #shellcheck source=src/env/bin/activate
-  source "${SCRIPT_DIR}"/env/bin/activate
-  python3 "${SCRIPT_DIR}"/pressCalendar/velo_detect.py
+  source ${SCRIPT_DIR}/env/bin/activate
+  python3 ${SCRIPT_DIR}/pressCalendar/velo_detect.py
 }
 
 # Main Script Handler
 script_Main() {
   while [ "${CURRENT_SCRIPT_STATE}" != "none" ]; do
-    case $USER_IS_ROOT in
+  echo "${CURRENT_SCRIPT_STATE}"
+    case ${USER_IS_ROOT} in
     true)
-      echo "Script should not be run as 'root' user"
-      echo "Exiting script"
-      CURRENT_SCRIPT_STATE="none"
+      case "${CURRENT_SCRIPT_STATE}" in
+      dev)
+	echo "${MIC_address}"; echo "${SPEAKER_address}"; echo "${CLIENT_SECRET_PATH}"
+	CURRENT_SCRIPT_STATE="none"
```

Comment:
dev test functions should be cleared before merging. Will handle on master branch